ospfd/ospf_interface.h

```diff
@@ -186,6 +186,7 @@ struct ospf_interface {
 
 	struct prefix *address;      /* Interface prefix */
 	struct connected *connected; /* Pointer to connected */
+	uint8_t passive_interface; /* 1 = passive interface, 0 = active interface */
```

Comment:
Could you clarify why we need this new variable when we already have `params->passive_interface` and the `OSPF_IF_PASSIVE_STATUS` macro?